Bindings/Java/Matlab/Hopper_Device/BuildHopper.m

```diff
@@ -24,26 +24,29 @@
 % Build a model of a one-leg hopper, with one muscle.
 
 p = inputParser();
-defaultMuscleModel = 'Thelen2003';
+defaultMuscleModel = 'Millard2012Equilibrium';
 defaultMaxIsometricForce = 4000.0;
 defaultMillardTendonParams = [0.049 28.1 0.67 0.5 0.25];
-defaultActivation = [0.0 1.99 2.0 3.89 3.9 4.0;
+defaultExcitation = [0.0 1.99 2.0 3.89 3.9 4.0;
                      0.3 0.3  1.0 1.0  0.1 0.1];
-defaultPrintModel = false; 
+defaultPrintModel = false;
+defaultAdditionalMass = 0;
```

Comment:
Given the structure of all this, it makes sense to do all of these types of changes within this function. I know I mentioned before to @nickbianco that it would be slick to generate a model upon loading the GUI and making changes to that one model, but the flow (of building a new model each time) is fine for this example and not worth changing.